src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -331,7 +337,8 @@
 short-summary: Show an image builder template.
 examples:
   - name: Show an image builder template (autogenerated)
-    text: az image template show --name mytemplate --resource-group my-group
+    text: |
+        az image template show --name mytemplate --resource-group my-group
```

Comment:
The name pattern for the example value here is different from others. If it's autogenerated, they could make the pattern consistent. For example the value for `--resource-group` would always be `MyResourceGroup`.

Reply:
Great!

I guess Aladdin leart from our wrong example. We provide him/her a wrong example years ago. :)

![image](https://user-images.githubusercontent.com/14357159/72147559-bf61d100-33d9-11ea-8269-3f10bfdef0f7.png)

Reply:
Manually fixed. Should infom Aladdin to tell them use Upper Camel Case style to set option value.